homeassistant/components/xiaomi_ble/sensor.py

```diff
@@ -153,7 +165,7 @@ def sensor_update_to_bluetooth_data_update(
                 (description.device_class, description.native_unit_of_measurement)
             ]
             for device_key, description in sensor_update.entity_descriptions.items()
-            if description.native_unit_of_measurement
+            if description.device_class
```

Comment:
There are entity descriptions without device class, how are these affected?

Reply:
They are not affected, see for example the formaldehyde test, which still runs fine and does not have a device class. The `description.device_class` is not the Home Assistant device class, but it is the BTHome device class. In most cases these are the same, but the BTHome device class `formaldehyde` is translated to device_class None for the HA entity.

Reply:
Good to know, thanks!